### ISSUE TITLE
Unable to format int/bigint field

### ISSUE THREAD

**Issue by chicoralves** [relferreira/metabase-sparksql-databricks-driver#7]
We can't format any integer or bigint field from SQLEndpoint table. The numbers appears with no decimal or thousand separator on our dashboard or data grid. Does have some release in order to fix it ?
Thanks in advance.

**Comment by kevinszczpanski** [comment]
Great Issue!

**Comment by bjgbeelen** [comment]
We experience the same thing

**Comment by relferreira** [comment]
Hello, sorry for the delay.
I'm a little busy now, but I will look into this problem as soon as possible.
I appreciate your patience.

**Comment by bjgbeelen** [comment]
Just sharing my steps here, as I am not able to continue working on it for now.
Just so others know I'm working on identifying this issue and trying to implement a fix (with zero Clojure knowledge)

I think I'm in the right direction, trying the following change:
```diff
+++ b/src/metabase/driver/hive_like.clj
@@ -55,7 +55,7 @@
     #"interval"         :type/*
     #"array.*"          :type/Array
     #"map"              :type/Dictionary
-    #".*"               :type/*))
+    #".*"               :type/Integer))
```

Suddenly in Metabase via databricks:

![image](https://user-images.githubusercontent.com/8626944/209150071-0783ebd3-da37-46f1-b3ec-c042476d6153.png)

This is of course not the solution yet, but at least it helps in identifying where the problem occurs

**Comment by relferreira** [comment]
Sorry for the delay. I'm having trouble reproducing your example @bjgbeelen.

I created the same table in Databricks:

![image](https://user-images.githubusercontent.com/4078815/209163170-2d2dcb04-4c8d-43a4-8eb1-edf338b4bb2d.png)

And got the following formatted result:

![image](https://user-images.githubusercontent.com/4078815/209162624-a25ac867-1461-4dcf-ad71-5771c22a0a2c.png)

But I might be doing something wrong.
Thank you for your initial investigation

**Comment by bjgbeelen** [comment]
Thanks @relferreira , interesting this is working for you out of the box!
I'm wondering what it could be. You are using the latest version of the driver that you published?

I'm curious about your setup.

What databricks runtime?
what metabase version?
Latest version of the driver that you published?

I'll continue investigating anyway, cause I think the original creator of this issue and others experience the same as me. 
But I'm surprised it's not happening for you, so I expect a certain combination of versions to be the cause?

**Comment by relferreira** [comment]
Hi @bjgbeelen, I'm running the container image defined on the project's Dockerfile

I've used:

Databricks runtime: SQLEndpoint on Azure
Metabase Version: v0.43.4
Latest Version published v1.4.0 https://github.com/relferreira/metabase-sparksql-databricks-driver/releases/tag/1.4.0

**Comment by relferreira** [comment]
I've tested it in Metabase v0.45.1, and it seems to work too...
Nevertheless, I uploaded a new release with an upgrade to the Databricks JDBC driver. Can you try this newer version?

Thanks for your help again

**Comment by bjgbeelen** [comment]
Hi @relferreira ,

thanks for responding so quickly. I was gone for a couple of days for xmas, but back now to solve this puzzle :-)
I tried your latest release but it doesn't make a difference. I tried debugging by printing the `database-type` from the `defmethod sql-jdbc.sync/database-type->base-type` method in `hive_like.clj` and it returned the type with capitals (`:INT`) and I think that's (related to) the issue, in that the regular expressions match the text case sensitive.
What I also don't understand yet is why that semicolon appears when printing the `database-type`

I tried applying lowercasing of the `database-type`, but it does not seem to work yet. I guess I don't understand the exact way that function works

```clojure
(defmethod sql-jdbc.sync/database-type->base-type :hive-like
  [_ database-type]
  (condp re-matches (name (str/lower-case database-type))
    #"boolean"          :type/Boolean
    #"tinyint"          :type/Integer
    #"smallint"         :type/Integer
    #":int.*"           :type/Integer
    #"int.*"            :type/Integer
    #"bigint"           :type/BigInteger
    #"float"            :type/Float
    #"double"           :type/Float
    #"double precision" :type/Double
    #"decimal.*"        :type/Decimal
    #"char.*"           :type/Text
    #"varchar.*"        :type/Text
    #"string.*"         :type/Text
    #"binary*"          :type/*
    #"date"             :type/Date
    #"time"             :type/Time
    #"timestamp"        :type/DateTime
    #"interval"         :type/*
    #"array.*"          :type/Array
    #"map"              :type/Dictionary
    #".*"               :type/*))
```

**Comment by bjgbeelen** [comment]
Is fixed in #10 so issue can be closed
